clients/bot_client.go

```diff
@@ -314,5 +314,5 @@ func (botClient *BotClient) ForwardRoomKeyToDevice(userID id.UserID, deviceID id
 		},
 	}
 
-	return botClient.olmMachine.SendEncryptedToDevice(device, forwardedRoomKey)
+	return botClient.olmMachine.SendEncryptedToDevice(device, mevt.ToDeviceForwardedRoomKey, forwardedRoomKey)
```

Comment:
I'm not sure this is the right event type actually, or if this should be `ToDeviceRoomKey`. I went with `ToDeviceForwardedRoomKey` due to the `forwardedRoomKey` variable.

Reply:
It's correct, the old version only allowed sending forwarded room key events (and I forgot to mention the parameter being added in changelogs, now specified in the [v0.9.7](https://github.com/tulir/mautrix-go/releases/tag/v0.9.7) notes)